### PR TITLE
Split api routes declaration

### PR DESCRIPTION
**Pull request by ael-code**
Cause it's not possible to use nested blueprint (mitsuhiko/flask#593) I've implemented a simple pattern to split api routes declaration. From now on every api groups will reside in it's own module.

This will make easier/cleaner to implement users api.

### CODE REVIEW

**Comment by boyska** [comment]
I'll open an issue to reimplement all of Flask. Be quick, please. :P

File: webant/api/blueprint_api.py [hunk 1]
```diff
@@ -1,34 +1,10 @@
...
-
-
-def make_success_response(message, http_code=200):
-    response = jsonify({'code': http_code, 'message': message})
-    response.status_code = http_code
-    return response
-
+from flask import Blueprint, current_app, jsonify
+from webant.util import add_routes
+from archivant_api import routes
```
Comment (boyska) [inline webant/api/blueprint_api.py]: this part is slightly counterintuitive: all this pull request is created so to make multiple modules adding routes to a single blueprint easier. But if you do `from foo import routes` you can only do it one time.
So probably
```python
import archivant_api

add_routes(api, archivant_api.routes)
```

would be easier

Comment (ael-code) [inline webant/api/blueprint_api.py]: On 10/04/2015 01:26 PM, BoySka wrote:
> In webant/api/blueprint_api.py
> <https://github.com/insomnia-lab/libreant/pull/191#discussion_r41096841>:
> 
>> -        self.err_code = err_code
>> -        self.message = message
>> -        self.details = details
>> -
>> -    def __str__(self):
>> -        return "http_code: {}, err_code: {}, message: '{}', details: '{}'".format(self.http_code, self.err_code, self.message, self.details)
>> -
>> -
>> -def make_success_response(message, http_code=200):
>> -    response = jsonify({'code': http_code, 'message': message})
>> -    response.status_code = http_code
>> -    return response
>> -
>> +from flask import Blueprint, current_app, jsonify
>> +from webant.util import add_routes
>> +from archivant_api import routes
> 
> this part is slightly counterintuitive: all this pull request is created
> so to make multiple modules adding routes to a single blueprint easier.
> But if you do |from foo import routes| you can only do it one time.
> So probably
> 
> import archivant_api
> 
> add_routes(api, archivant_api.routes)
> 
> would be easier

There are at least two solution to the problem you've raised:
 - use the `as` keyword to differentiate the imported `routes` objects
(used by me in `users_api` branch)
 - the one that you suggested, import all module and specify later.

I think this is a minor issue and we can change it at the next api
implementation. It's a simple import problem, but if you want to change
it now you need only to choose the solution you like most.

**Comment by ael-code** [comment]
In the meanwhile I've rebased to solve conflicts with latest commits on master branch

**Pull request merged**